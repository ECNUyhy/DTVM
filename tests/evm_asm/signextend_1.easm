// SIGNEXTEND(0x00, 0xFF) = 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF
PUSH1 0xFF
PUSH1 0x00  // Extend from 1 byte (0 means 1 byte)
SIGNEXTEND
