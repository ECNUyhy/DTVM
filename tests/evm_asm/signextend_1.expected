status: success
error_code: 0
stack:
  - 'FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF'
memory: ''
storage: {}
transient_storage: {}
return: ''
events: []
